e2e/e2e-scenario.sh

```diff
@@ -75,8 +75,9 @@ az vmss create -n ${VMSS_NAME} \
     --vm-sku $VM_SKU \
     --instance-count 1 \
     --assign-identity $msiResourceID \
-    --image "microsoft-aks:aks:aks-ubuntu-1804-2022-q1:2022.02.01" \
+    --image "MicrosoftCBLMariner:cbl-mariner:cbl-mariner-2-gen2:latest" \
     --upgrade-policy-mode Automatic \
+    --os-disk-size-gb 40 \
```

Comment:
got disk size errors, but I thought default is 30gb which should be identical to vanilla...I guess the base image is maybe small by default with marketplace?